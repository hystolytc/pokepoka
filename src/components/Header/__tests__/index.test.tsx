import React from 'react'
import '@testing-library/jest-dom/extend-expect'
import { render } from '@testing-library/react'
import { Header } from '../index'

const describeComponent = '<Header /> test case'

describe(describeComponent, () => {
  const { container } = render(
    <Header>
      <p>Header Test</p>
    </Header>
  )

  test(`Render ${describeComponent} correctly`, () => {
    expect(container).toMatchSnapshot()
  })
})