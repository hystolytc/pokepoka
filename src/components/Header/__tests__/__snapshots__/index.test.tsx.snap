// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> test case Render <Header /> test case correctly 1`] = `
<div>
  <header
    class="css-1vo74qe"
  >
    <p>
      Header Test
    </p>
  </header>
</div>
`;
